cmd/clusterctl/clusterdeployer/bootstrap/options.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bootstrap
+
+import "github.com/spf13/pflag"
+
+type Options struct {
+	Type             string
+	Cleanup          bool
+	MiniKubeFlags    []string
+	MinikubeVmDriver string
+	KubeConfig       string
+}
+
+func (o *Options) AddFlags(fs *pflag.FlagSet) {
+	fs.StringVarP(&o.Type, "bootstrap-type", "", "minikube", "The cluster bootstrapper to use")
+	fs.BoolVarP(&o.Cleanup, "bootstrap-cluster-cleanup", "", true, "Whether to cleanup the bootstrap cluster after bootstrap")
+	fs.StringVarP(&o.KubeConfig, "bootstrap-cluster-kubeconfig", "e", "", "Sets the bootstrap cluster to be an existing Kubernetes cluster.")
+	fs.StringSliceVarP(&o.MiniKubeFlags, "minikube-flags", "", []string{}, "Command line flags to be passed to minikube")
+	fs.StringVarP(&o.MinikubeVmDriver, "minikube-vm-driver", "", "", "Adds --vm-driver flag to minikube, for more information refer to https://github.com/kubernetes/minikube/blob/master/docs/drivers.md")
```

Comment:
so this is specific to minikube, can we handle it using the above proposed `boostrap-flags`?
i personally don't see the benefit to "wrap" the `--vm-driver` flag in such a way.

Reply:
+1 on not wrapping specific flags to minikube. If there is something we need to change about how we manage minikube I could see a flag for that, but doing a passthrough doesn't seem particularly useful.

Reply:
Fixed!